### PR TITLE
[sailfish-office] Open csv files as spreadsheet documents. Contributes JB#46745

### PR DESCRIPTION
**Pull request by mphilippov-omp**
Make csv files visible in document list

### CODE REVIEW

File: models/documentlistmodel.cpp [hunk 1]
```diff
@@ -191,7 +192,8 @@ int DocumentListModel::mimeTypeToDocumentClass(QString mimeType) const
                || mimeType == QLatin1String("application/vnd.openxmlformats-officedocument.presentationml.slideshow")
                || mimeType == QLatin1String("application/vnd.ms-powerpoint.presentation.macroEnabled.12")
                || mimeType == QLatin1String("application/vnd.ms-powerpoint.template.macroEnabled.12")
-               || mimeType == QLatin1String("application/vnd.ms-powerpoint.slideshow.macroEnabled.12") ) {
+               || mimeType == QLatin1String("application/vnd.ms-powerpoint.slideshow.macroEnabled.12") 
+               || mimeType == QLatin1String("text/csv")) {
```
Comment (pvuorela) [inline models/documentlistmodel.cpp]: Presentation?

File: models/documentlistmodel.h [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright (C) 2019 Open Mobile Platform LLC
```
Comment (pvuorela) [inline models/documentlistmodel.h]: No actual changes here?

File: models/trackerdocumentprovider.h [hunk 3]
```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright (C) 2019 Open Mobile Platform LLC
```
Comment (pvuorela) [inline models/trackerdocumentprovider.h]: No changes here?

Comment (mphilippov-omp) [reply models/trackerdocumentprovider.h]: Changes are in source file

File: models/documentlistmodel.h (hunk 2 above)
Comment (mphilippov-omp) [reply models/documentlistmodel.h]: In source file

File: models/documentlistmodel.cpp (hunk 1 above)
Comment (mphilippov-omp) [reply models/documentlistmodel.cpp]: No. I added "text/csv"

File: models/documentlistmodel.h (hunk 2 above)
Comment (mphilippov-omp) [reply models/documentlistmodel.h]: I removed our copyright from here

File: models/trackerdocumentprovider.h (hunk 3 above)
Comment (mphilippov-omp) [reply models/trackerdocumentprovider.h]: I removed it from here

File: models/documentlistmodel.cpp (hunk 1 above)
Comment (pvuorela) [reply models/documentlistmodel.cpp]: Be it however, you added csv to be handled as a presentation format.

Comment (mphilippov-omp) [reply models/documentlistmodel.cpp]: Ooops, I see. I'll correct

Comment (mphilippov-omp) [reply models/documentlistmodel.cpp]: Done

File: models/documentlistmodel.cpp [hunk 4]
```diff
@@ -199,7 +200,8 @@ int DocumentListModel::mimeTypeToDocumentClass(QString mimeType) const
                || mimeType == QLatin1String("application/vnd.openxmlformats-officedocument.spreadsheetml.template")
                || mimeType == QLatin1String("application/vnd.ms-excel.sheet.macroEnabled")
                || mimeType == QLatin1String("application/vnd.ms-excel.sheet.macroEnabled.12")
-               || mimeType == QLatin1String("application/vnd.ms-excel.template.macroEnabled.12") ) {
+               || mimeType == QLatin1String("application/vnd.ms-excel.template.macroEnabled.12") 
+               || mimeType == QLatin1String("text/csv")) ) {
```
Comment (pvuorela) [inline models/documentlistmodel.cpp]: Doesn't compile. Too many parenthesis.

Comment (mphilippov-omp) [reply models/documentlistmodel.cpp]: I see. Done

**Comment by pvuorela** [comment]
LGTM

**Comment by mphilippov-omp** [comment]
Will it be merged?

**Comment by mphilippov-omp** [comment]
> LGTM

will it be merged?

**Pull request merged**